src/record/list.js

```diff
@@ -172,10 +172,13 @@ List.prototype.subscribe = function() {
 	}
 
 	//Make sure the callback is invoked with an empty array for new records
-	parameters.callback = function( callback ) {
+	var listCallback = function( callback ) {
 		callback( this.getEntries() );
 	}.bind( this, parameters.callback );
 
+	parameters.callback.wrappedCallback = listCallback;
+	parameters.callback = listCallback;
```

Comment:
This looks weird:
- setting a property of an object
- overwriting the whole object

Maybe create two variables for `_record. subscribe` and `_record.unsubscribe` instead of modifying the `parameters` variable.